docs/configure/environment-variables.mdx

```diff
@@ -72,7 +72,7 @@ Then you can access this environment variable anywhere, even within your stories
   You can also use specific files for specific modes. Add a `.env.development` or `.env.production` to apply different values to your environment variables.
 </Callout>
 
-You can also pass these environment variables when you are [building your Storybook](../sharing/publish-storybook.mdx) with `build-storybook`.
+You can also pass these environment variables when you are [building your Storybook](../sharing/publish-storybook.mdx) with `storybook build`.
```

Comment:
Same goes here. In this context we're aiming for the script as well

Reply:
This is actually the page and the line that confused me as a passive Storybook user, and kicked off opening this whole PR 😂 
I will make all the changes you have suggested, but maybe there is another improvement that we can make that can be explicit about when `build-storybook` is available and when it isn't, because reading the migration guide makes it seem like it is not available anymore https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#start-storybook--build-storybook-binaries-removed

Reply:
Thanks for pointing that out. We can improve that portion of the documentation to avoid any confusion you or any other Storybook user may have.